README.md

```diff
@@ -172,20 +172,20 @@ This is beyond the scope of this project, however please see https://github.com/
 One of the inspirations for writing this library was to create a simple to use interface for creating, reading, writing, verifying, and linting BagIt specification bags. The coding therefore tries to adhere with the best practices in [Effective Java by Joshua Bloch](https://www.amazon.com/Effective-Java-Joshua-Bloch/dp/0134685997) as well as experiences from the team members.
 
 #### Classes/Methods not to use outside this project
-There are many classes that were not designed to be used outside this project, the rules for this are
-* If a class/method does not contain a javadoc
-* If a class is in a package named "internal"
-* If a class is final, that class was not designed to be extended by users outside this project
+There are many classes that were not designed to be used outside this project, the rules for this are:
+* If a class/method does not contain a javadoc.
+* If a class is in a package named "internal".
+* If a class is final, that class was not designed to be extended by users outside this project.
 
-#### javadocs
+#### Javadocs
 All public interfaces/classes have javadocs detailing what the class's responsibilities are and, what the methods do and are used for. 
 TODO - add link to http://www.javadoc.io/doc
```

Comment:
We can remove the TODO here since the link was added in the badge at the top of the document. We should also leave the link here for convenience.